src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND = 10;
+    private static final int BOUND_OF_CASES = 10;
```

Comment:
The constant BOUND_OF_CASES is misleading since it implies there are 10 cases, but the switch statement only has 5 cases. Consider renaming it to match the actual number of figure types available, or adjust the switch cases to match this number.